src/routes/stream_routes/add.tsx

```diff
@@ -45,7 +46,10 @@ export const StreamRouteAddForm = (props: Props) => {
 
   const postStreamRoute = useMutation({
     mutationFn: (d: StreamRoutePostType) =>
-      postStreamRouteReq(req, pipeProduce()(d)),
+      postStreamRouteReq(
```

Comment:
What type of changes are outside the `e2e` directory? Is it a fix for a discovered bug when writing test cases? If so, the fix PR should be separated.

Reply:
done. #3117